Tests/Routing/Loader/RestYamlCollectionLoaderTest.php

```diff
@@ -27,6 +27,36 @@
 class RestYamlCollectionLoaderTest extends LoaderTest
 {
     /**
+     * Test that YAML file is empty.
+     */
+    public function testLoadDoesNothingIfEmpty()
+    {
+        $collection = $this->loadFromYamlCollectionFixture('empty.yml');
+        $this->assertEquals([], $collection->all());
+    }
+
+    /**
+     * Test that YAML files are invalid.
+     *
+     * @expectedException \InvalidArgumentException
+     * @dataProvider getPathsToInvalidFiles
+     */
+    public function testLoadThrowsExceptionWithInvalidFile($filePath)
```

Comment:
i can't add `@expectedExceptionMessage` because of i'm using `@dataProvider` so the exception message may be difference for every tests file.
for example:
when exception catcby ParseException, the exception message is 'The file "/home/your/path/bad_format.yml" does not contain valid YAML' [RestYamlCollectionLoader.php](https://github.com/indragunawan/FOSRestBundle/blob/f2a557ac36284516c319e7925546907a6ab6eeee/Routing/Loader/RestYamlCollectionLoader.php#L67)
when the value of the routes file is not array, the exception message is 'The file "/home/your/path/nonvalid2.yml" must contain a YAML array.' [RestYamlCollectionLoader.php](https://github.com/indragunawan/FOSRestBundle/blob/f2a557ac36284516c319e7925546907a6ab6eeee/Routing/Loader/RestYamlCollectionLoader.php#L80)

i need to split the test to use `@expectedExceptionMessage`, how about this?